crypto/pkcs7/pkcs7.c

```diff
@@ -1250,7 +1252,9 @@ static BIO *pkcs7_data_decode(PKCS7 *p7, EVP_PKEY *pkey, X509 *pcert) {
       if (!pkcs7_cmp_ri(ri, pcert)) {
         break;
       }
+#if !defined(BORINGSSL_UNSAFE_FUZZER_MODE)
```

Comment:
now that we don't specify `pcert` when fuzz testing, can we drop this macro guard?